lib/persisted-model.js

```diff
@@ -770,7 +773,7 @@ module.exports = function(registry) {
 
     setRemoting(PersistedModel, 'deleteById', {
       aliases: ['destroyById', 'removeById'],
-      description: 'Delete a model instance by id from the data source.',
+      description: g.f('Delete a model instance by id from the data source.'),
```

Comment:
not confidence should we use {{id}} here and the other two lines i commented above lol. feel free to ignore it if unnecessary.